### PR TITLE
chore: add default Travis config

### PR DESCRIPTION
**Pull request by DanielRuf**
We should probably prepare some real unit tests =)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - "8"
+  - "10"
```
Comment (developit) [inline .travis.yml:4]: could just go with node_js: 10 I think

**Comment by developit** [comment]
Agreed Re: tests! I'd like to set them up using [Karmatic](https://github.com/developit/karmatic) so they run in a real DOM (rather than Jest).

File: .travis.yml (hunk 1 above)
Comment (DanielRuf) [reply .travis.yml:4]: In my opinion we should test on current LTS and stable.

Comment (developit) [reply .travis.yml:4]: What for though? This module doesn't run in Node, it runs in Browsers - would make more sense to test on a couple different browsers, Node is just running the test runner.

Comment (DanielRuf) [reply .travis.yml:4]: Sure, bit keep in mind that also the testrunner (see the issue at karmatic) may fail under a different Node.js version and contributors (developers) also need to run the tests.

**Review by developit — APPROVED** [review]

**Pull request merged**